Vue InstantSearch/query-suggestions/src/App.vue

```diff
@@ -13,55 +13,56 @@
     <div class="container">
       <ais-instant-search
         :search-client="searchClient"
-        index-name="instant_search_demo_query_suggestions"
+        index-name="instant_search"
```

Comment:
this does a useless query at root, since the autocomplete index should _not_ have the disjunctive facets set by the root / now sibling.

Reply:
I could filter the main index out with a custom search client, but not sure that's the smartest idea to put in a guide